WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -894,7 +894,15 @@ public static void setSupportEmail(String email) {
     }
 
     public static String getSupportEmail() {
-        return getString(DeletablePrefKey.SUPPORT_EMAIL);
+        String email = getString(DeletablePrefKey.SUPPORT_EMAIL);
+        // Zendesk can't create support tickets for Automattic email addresses from
+        // those with a staff member role (admin, agent, etc.), so insert "-testing"
+        // into the email to make it work
+        if ((email.contains("@automattic.com") || email.contains("@a8c.com")) && !email.contains("-testing@")) {
+            return email.replace("@", "-testing@");
```

Comment:
I think we are suppose to add an `+testing`, not `-`? /cc @jkmassel

Reply:
@crazytonyli Thanks for catching that! I've made the correction.

Reply:
@crazytonyli Are we okay to merge this or are we waiting for @jkmassel to approve it?